thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/notification/commons/JiraEntity.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.notification.commons;
+
+import java.util.List;
+
+
+/**
+ * ThirdEye's Jira Settings Holder
+ */
+public class JiraEntity {
+  private String jiraProject;
+  private Long jiraIssueTypeId;
+  private String summary;
+  private String description;
+  private String assignee;
+  private List<String> labels;
+  private long mergeGap;
```

Comment:
Instead of being part of the `JiraEntity`, shouldn't the `mergeGap` be part of the config like the merger we are using in detection?.